### DISCUSSION TITLE
Arrange feature as move to back / move to front

### DISCUSSION THREAD

**Discussion by ilkerbesken** [xournalpp/xournalpp#4200] [Ideas]
Hi,
Many other note-taking and drawing applications have the "Arrange" feature, which is very helpful in quickly locating (z-index changing) objects. Xournalpp already works with Layers, but copying and pasting between layers reduces usability. I wonder if a quick "Arrange" shortcut could be added?
If such a feature already exists and I couldn't find it, I'd appreciate it if you could help.
I'm sorry if this topic has been brought up before, I opened the topic because I couldn't find it when I searched.

Thanks

for example:
![Adsız01](https://user-images.githubusercontent.com/26311380/184711028-e19a7105-dc23-4b7e-9bc3-8f94b69b0e84.png)

**Comment by rolandlo** [comment]
There is already menu `Edit > Arrange`. If you can't find it there, upgrade to the [nightly release](https://github.com/xournalpp/xournalpp/releases/tag/nightly)

**Reply by ilkerbesken** [reply]
Hi, it works with stable v1.1.1. too 
Thanks alot....
